tripy/tripy/frontend/module/module.py

```diff
@@ -203,6 +203,15 @@ def find_module(module: Union[Module, List, Dict], sub_strs: List[str]):
                     module = operator.attrgetter(child_name)(module)
             return module
 
+        expected_keys = set(self.state_dict())
+        provided_keys = set(state_dict)
+        missing_keys = expected_keys - provided_keys
+        unexpected_keys = provided_keys - expected_keys
+        if len(missing_keys) > 0:
+            logger.warning("Missing key(s) in state_dict: {}. ".format(", ".join(f'"{k}"' for k in missing_keys)))
+        if len(unexpected_keys) > 0:
+            logger.warning("Unexpected key(s) in state_dict: {}. ".format(", ".join(f'"{k}"' for k in unexpected_keys)))
```

Comment:
Should we use `raise_error` instead?
```suggestion
        if missing_keys:
            logger.warning("Missing key(s) in state_dict: {}. ".format(", ".join(missing_keys)))
        if unexpected_keys:
            logger.warning("Unexpected key(s) in state_dict: {}. ".format(", ".join(unexpected_keys)))
```

Reply:
Seems torch will raise an error, I'll use raise_error too.

Reply:
Added a `strict` option, similar as torch. If strict is True (which is by default), error will be raised. Otherwise the `load_state_dict` function will only load the valid keys, and return the `missing_keys, unexpected_keys`.